components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/certificatemgt/CertificateManager.java

```diff
@@ -22,6 +22,7 @@
 import org.wso2.carbon.apimgt.api.dto.CertificateMetadataDTO;
 import org.wso2.carbon.apimgt.api.dto.ClientCertificateDTO;
 import org.wso2.carbon.apimgt.api.model.APIIdentifier;
+import org.wso2.carbon.apimgt.api.model.APIProductIdentifier;
```

Comment:
Seems this import is not used. Shall we remove it?

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/9604/commits/77531e9f3feda9420f15f6cd639af122781038dd